### PR TITLE
Add documentation how to use ecpp (Fixes #29)

### LINKED ISSUE #29: Document usage
We should have a manual that explains:
 - [ ] which dependencies are necessary to install
 - [ ] how to use ecpp to verify a moduli file
 - [ ] how to generate new primality certificates

### PR DESCRIPTION
**Pull request by Jakuje**
Initial draft of basic documentation written during my attempts to use the tool :) Comments, suggestions and improvements welcomed.

### CODE REVIEW

**Comment by tomato42** [comment]
we also need documentation how to use `ecpp` to verify the primality certificate

when I have many certs to verify I usually use the gnu parallel to do that; it's fairly easy to do that for newly created certificates (i.e. run it for all .out files in the `in/` directory)

**Comment by Jakuje** [comment]
I added the OpenSSH 8.3p1 certificates that I managed to generate over the weekend and a step to verify certificates (which I am running at this moment).

File: README.md [hunk 1]
```diff
@@ -9,3 +9,72 @@ primality proofs, commonly generated by the [Primo](http://www.ellipsa.eu/public
...
+in this example using 16 parallel processes:
+
+    parallel -j16 "echo {} && ./ecpp -i {}" ::: in/*.out
+
+Now, we can add the primality certificates to `certificates/` directory
+
+## Verifying primes in OpenSSH moduli file again
+
+Running `ecpp` again as in the first example should give us certificate for
+each prime in the moduli file, confirming these are really primes.
```
Comment (tomato42) [inline README.md:78]: technically, this confirms that we have certificates for all primes in the moduli file

Comment (Jakuje) [reply README.md]: Thanks. Fixed in current version.

**Comment by Jakuje** [comment]
OpenSSH 8.3p1 certs added as #31. Squashed this one to limit changes only to documentation changes.

**Comment by tomato42** [comment]
looks good, thank you!

**Pull request merged**